cita-executor/src/backlogs.rs

```diff
@@ -494,19 +494,21 @@ mod tests {
     #[test]
     fn test_backlog_is_completed_with_none() {
         let height = 9;
+        let path = "test-rocksdb/backlog_is_completed_with_none";
```

Comment:
It would be better to define a const variable point to `test-rocksdb`

Reply:
yeah. 
btw, It'll be useless in the future

```
// TODO Use MemoryDB instead of RocksDB.
```